tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary.feature

```diff
@@ -27,6 +27,7 @@ Scenario: 1. Get the order summary
         | catalogue-solutions | incomplete | 0     |
         | additional-services | incomplete | 0     |
         | funding-source      | incomplete |       |
+    And the order Section Status is incomplete

```

Comment:
Would it be possible to assert the section status as part of one scenario? By adding this statement to each of the tests it makes them a little brittle i.e. if something changes with the logic surrounding the section status it will potentially break all the tests in this file rather than just one. What do you think?